docs/content/tutorial/step_15.ngdoc

```diff
@@ -0,0 +1,327 @@
+
+* Lack of information about the current results when the user interacts with the filters.
+* Lack of labels elements in the search and order filters in the Phone list.
+* Lack of headers elements to help the user to navigate throw the content of the page.
+* In the Phone detail, the user can't access via keyboard over the thumbnails images.
+
+In the next sections we are going to show you, how to fix some of these issues to make the application accessible.
+
+To help us, we will use the {@link ngAria ngAria} module, and we will install the [protractor-a11y-plugin] plugin too.
+
+## Dependencies
+
+The `ngAria` module, is distributed separately from the core Angular framework. 
+
+Since we are using [Bower][bower] to install client-side dependencies, this step updates the
+`bower.json` configuration file to include the new dependencies:
+
+<br />
+**`bower.json`:**
+
+```
+{
+  "name": "angular-phonecat",
+  "description": "A starter project for AngularJS",
+  "version": "0.0.0",
+  "homepage": "https://github.com/angular/angular-phonecat",
+  "license": "MIT",
+  "private": true,
+  "dependencies": {
+    "angular": "1.5.x",
+    "angular-animate": "1.5.x",
+    "angular-aria": "1.5.x",
+    "angular-mocks": "1.5.x",
+    "angular-resource": "1.5.x",
+    "angular-route": "1.5.x",
+    "bootstrap": "3.3.x",
+    "jquery": "2.2.x"
+  }
+}
+```
+
+* `"angular-aria": "1.5.x"` tells bower to install a version of the angular-aria module that
+  is compatible with version 1.5.x of Angular.
+  
+To install the [protractor-a11y-plugin], we can add the plugin into the `devDependencies` section in the `package.json` file,
+
+<br />
+**`package.json`:**
+
+```
+{
+  "devDependencies": {
+    "bower": "^1.7.7",
+    "http-server": "^0.9.0",
+    "jasmine-core": "^2.4.1",
+    "karma": "^0.13.22",
+    "karma-chrome-launcher": "^0.2.3",
+    "karma-firefox-launcher": "^0.1.7",
+    "karma-jasmine": "^0.3.8",
+    "protractor": "^3.2.2",
+    "protractor-accessibility-plugin": "^0.1.1",
+    "shelljs": "^0.6.0"
+   }
+}
+```
+
+Now, we must tell bower to download and install these dependencies.
+
+```
+npm install
+```
+
+<div class="alert alert-info">
+  **Note:** If you have bower installed globally, you can run `bower install`, but for this project
+  we have preconfigured `npm install` to run bower for us.
+</div>
+
+<div class="alert alert-warning">
+  **Warning:** If a new version of Angular has been released since you last ran `npm install`, then
+  you may have a problem with the `bower install` due to a conflict between the versions of
+  angular.js that need to be installed. If you run into this issue, simply delete your
+  `app/bower_components` directory and then run `npm install`.
+</div>
+
+
+## How to use the `ngAria` module
+
+The `ngAria` module provides support for common ARIA attributes that convey state or semantic information about the application for users of assistive technologies, such as screen readers.
```

Comment:
We should also have a paragragh on how our specific app will be benefited just by including `ngAria` (to give more emphasis on what `ngAria` does under the hood for us).